### PR TITLE
[Modular] Removes Rusting from SSDecay

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Removes Rust from SSDecay. That's it. Everything else is untouched, but Rust is entirely removed from the system.

Also fixes tram tiles breaking/organizes the tile handling file

- [ ] Little side-note that I'd like to ask; Maintainers, should I remove the Nest portion of SSDecay while I'm here too? It's entirely unused and probably can just be tossed, it doesn't seem very fitting to the server.

## How This Contributes To The Nova Sector Roleplay Experience
I've talked with people about this a lot before the schism. Especially engineers _(which is a title I fall under. #saltpr)_

Nobody likes the rust.
It's a hassle to deal with - removal always has timers, always has to be done a maximum of 5 tiles at a time, and typically involves cutting holes through walls and moving machines to reach it all.
So, it's left untouched. Every time. Everybody's stopped interacting with the system at all, as it amounts to nothing but busywork.

The filth stays. We love the filth. Janitorial _(also a title I fall under)_ LOVES the filth.
But the rust is pain. Also still extremely buggy and problematic I've seen it on so much titanium ughhhh.

## Proof of Testing

<details>
<summary>Screenshots/Videos</summary>
  
image

Still nice and filthy. Just no more rust.
image

</details>

## Changelog

:cl:
qol: After complaints of excess rusting, CC has finally installed some dehumidifiers in their stations! (SSDecay will no longer rust tiles)
fix: Due to new maintenance measures, the Tram rail-line should no longer rarely start the shift with damage. (SSDecay will no longer destroy Tram tiles)
/:cl:

### CODE REVIEW

**Comment by OrionTheFox** [comment]
> You currently have a negative Fix/Feature pull request delta of -36. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

SHUT UP BOT GRR

File: modular_nova/modules/decay_subsystem/code/decay_turf_handling.dm [hunk 1]
```diff
@@ -1,14 +1,19 @@
-/turf/open/floor
-	turf_flags = IS_SOLID | CAN_DECAY_BREAK_1 // We do it this way because we can then easily pick what we don't want to be broken.
-
 /turf/closed/wall
 	flags_1 = CAN_BE_DIRTY_1
+	// Allow walls to get filthy (Does TG really not already have this?)
```
Comment (OrionTheFox) [inline modular_nova/modules/decay_subsystem/code/decay_turf_handling.dm:3]: So, I was thinking on this -- does this even have relevance to SSDecay? The flag is for allowing a tile to be dirty on mapload, but like - It feels like the flag is just going to break something;
IIRC TG has an entire thing about certain tiles not having filth while mapping, and this tile being changed **to** have it... should I toss this line?

File: modular_nova/modules/decay_subsystem/code/decay_turf_handling.dm [hunk 2]
```diff
@@ -1,14 +1,19 @@
-/turf/open/floor
-	turf_flags = IS_SOLID | CAN_DECAY_BREAK_1 // We do it this way because we can then easily pick what we don't want to be broken.
-
 /turf/closed/wall
 	flags_1 = CAN_BE_DIRTY_1
+	// Allow walls to be dirty at round start
```
Comment (OrionTheFox) [inline modular_nova/modules/decay_subsystem/code/decay_turf_handling.dm:3]: So, I was actually looking at this line - is this even something that SSDecay uses? Should it even be a flag added still? Is this even doing anything at all?

IIRC TG had a thing to check for filth in invalid tiles on mapload, and if walls are one of those tiles - could this possibly be breaking it, or be at risk of breaking it?

File: modular_nova/modules/decay_subsystem/code/decay_turf_handling.dm [hunk 3]
```diff
@@ -1,14 +1,19 @@
...
 
 /turf/open/floor/glass
-	turf_flags = IS_SOLID /// No breaking the glass (doesn't leave plating behind)
+	turf_flags = IS_SOLID
+	// No breaking  glass (doesn't leave plating behind)
 
-/turf/open/misc/asteroid
-	turf_flags = NONE /// They shouldn't break and they shouldn't be dirty, it's literally already a dirty turf.
+/turf/open/floor/tram
+	turf_flags = IS_SOLID
```
Comment (GoldenAlpharex) [inline modular_nova/modules/decay_subsystem/code/decay_turf_handling.dm:18]: Why are the comments put on the line after? It just makes it less clear what the comment is about. Move them to the line before if you want to have them on a different line, but never on the line after.

**Review by GoldenAlpharex — DISMISSED** [review]
Other than that, though, I approve of this PR, I don't like rusted walls, they're a pain.

File: modular_nova/modules/decay_subsystem/code/decay_turf_handling.dm (hunk 3 above)
Comment (OrionTheFox) [reply modular_nova/modules/decay_subsystem/code/decay_turf_handling.dm]: It's 'cuz I did it at ungodly hours in the morning whoops
All fixed, it's either on the same line or the one above it

**Pull request merged**